### DISCUSSION TITLE
Change order of divs on grid resize

### DISCUSSION THREAD

**Discussion by ahron1** [pure-css/pure#1064] [General]
Say I have divs A and B next to each other on widescreen - A to the left, and B to the right.

On narrow screen, when it collapses to single column, it'll be A on top of B. 

Is it possible to achieve a different order? For example, on wide screen - A on the left and B on the right, but on narrow screen it should be B on top of A.

**Comment by redonkulus** [comment]
With the current grid classes no. But you can easily achieve this with flexbox (using the `order` property) or CSS grids.
